zeppelin-web/src/app/notebook/paragraph/paragraph-chart-selector.html

```diff
@@ -13,49 +13,55 @@
 -->
 
 <div id="{{paragraph.id}}_switch"
-     ng-if="(paragraph.result.type == 'TABLE' || apps.length > 0 || suggestion.available && suggestion.available.length > 0) && !asIframe && !viewOnly"
+     ng-if="(paragraph.result.type == 'TABLE' || paragraph.result.type == 'NETWORK' || apps.length > 0 || suggestion.available && suggestion.available.length > 0) && !asIframe && !viewOnly"
      class="btn-group"
      style="margin-bottom: 10px;">
 
   <button type="button" class="btn btn-default btn-sm"
-          ng-if="paragraph.result.type == 'TABLE'"
+          ng-if="paragraph.result.type == 'TABLE' || paragraph.result.type == 'NETWORK'"
```

Comment:
The goal of this PR is to full advantage the [property graph](https://github.com/tinkerpop/gremlin/wiki/Defining-a-Property-Graph) by the zeppelin platfom.
```
%spark
print(s"""
%network {
    "nodes" : [
        {"id" : 1, "label" : "User", "data" : { "fullname" : "Andrea Santurbano" }},
        {"id" : 2, "label" : "User", "data" : { "fullname" : "Moon soo Lee" }}
    ],
    "edges" : [{"source" : 2, "target" : 1, "id" : 1, "label" : "HELPS", "data" : { "project" : "Zeppelin", "githubUrl": "https://github.com/apache/zeppelin/pull/1582" } }]
}
""")
```
![neo4-property-graph](https://cloud.githubusercontent.com/assets/1833335/20022769/5bb39abc-a2dc-11e6-8647-57ff7a21e577.PNG)
This type of graph can be displayed in other ways like tabular, donut, etc...
![neo4j-donut-chart](https://cloud.githubusercontent.com/assets/1833335/20022908/165073cc-a2dd-11e6-92dc-3055d68c7392.PNG)
So i think the other visualizations must stay selectable.
Thanks for the feedback!